internal/app/cmd/cmd.go

```diff
@@ -103,7 +112,8 @@ func runHTTP(cmd *cobra.Command, cfg killgrave.Config) error {
 
 	<-done
 	if err := srv.Shutdown(); err != nil {
-		log.Fatal(err)
+		log.Error(err)
+		os.Exit(1)
```

Comment:
Same as above, and for the rest of calls to both: `log.Error`, and `os.Exit(1)`.